boxsdk/client/client.py

```diff
@@ -342,7 +352,11 @@ def create_user(self, name, login=None, **user_attributes):
             user_attributes['is_platform_access_only'] = True
         box_response = self._session.post(url, data=json.dumps(user_attributes))
         response = box_response.json()
-        return User(self._session, response['id'], response)
+        return Translator().translate(response['type'])(
```

Comment:
Can be `Translator().translate('user')` for consistency with other endpoints. Though this works too.

(The reason `get_shared_item()` above is special, is because `/shared_items` may return a file or folder.)